addon/mixins/in-viewport.js

```diff
@@ -91,14 +91,23 @@ export default Mixin.create({
       return;
     }
 
-    const $contextEl = $(context);
+    const contextBoundingClientRect = context.document.documentElement.getBoundingClientRect();
     const boundingClientRect = element.getBoundingClientRect();
+    let width, height;
+    if (!('width' in contextBoundingClientRect)) {
+      // IE8 and lower:
+      width = contextBoundingClientRect.right - contextBoundingClientRect.left;
+      height = contextBoundingClientRect.bottom - contextBoundingClientRect.top;
```

Comment:
@les2 I'm not too certain as I have no way to test IE on my machine. I feel like the final decision here should be left up to @poteto. 

I think that if this isnt a for sure solution, then the best thing might be to use `contextBoundingClientRect.{width,height}` and default back to the previous jQuery solution for older browsers.